src/snowflake/snowpark/ml/transformer.py

```diff
@@ -429,7 +434,7 @@ def transform(
         df: DataFrame,
     ) -> DataFrame:
         """
-        The ordinal encoder will be "null" if there are new types in dataframe fed into transform compared to dataframe
+        The encoder value will be null if there are unknown types in this dataframe compared to that used in fit()
```

Comment:
```suggestion
        The encoded value will be null if there are unknown types in this dataframe compared to that dataframe used in fit() function.
```
and remove the following line